lang/es.json

```diff
@@ -11,13 +11,13 @@
   "ITEM.TypeGear": "Equipo",
   "ITEM.TypeItemattachment": "Accesorio",
   "ITEM.TypeItemmodifier": "Modificador objeto",
```

Comment:
There's a few more here to convert

Reply:
😅 done

Reply:
Erm nope you reverted a few that were fine 😂 sorry...

This file `es.json` has a bunch to convert to the new format on lines [2-13](https://github.com/StarWarsFoundryVTT/StarWarsFFG/blob/d1dd24d1e8dc6f6f5aa4099d9e62075d9fe2c128/lang/es.json#L2-L13) and another on line [15](https://github.com/StarWarsFoundryVTT/StarWarsFFG/blob/d1dd24d1e8dc6f6f5aa4099d9e62075d9fe2c128/lang/es.json#L15).